packages/kit/src/runtime/server/page/actions.js

```diff
@@ -247,7 +256,24 @@ async function call_action(event, actions) {
 		);
 	}
 
-	return with_event(event, () => action(event));
+	return record_span({
+		name: 'sveltekit.action',
```

Comment:
`action` is vague. perhaps `sveltekit.form`?

Reply:
I went with `form_action` as `form` could be confused with remote functions